app/extensions/brave/locales/en-US/menu.properties

```diff
@@ -80,6 +80,14 @@ newSessionTab=New Session Tab
 newWindow=New Window
 reopenLastClosedTab=Reopen Last Closed Tab
 print=Print…
+emailPageLink=Email Page Link…
+tweetPageLink=Tweet Page…
+facebookPageLink=Share on Facebook…
```

Comment:
I think these would make more sense as the generic template, like "Share on {{siteName}}" and the the l10n library could have the service name input as args. This would prevent translators from possibly changing the usage of the name / brand / platform for the share